src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -143,6 +143,7 @@ protected function fillGithubActions()
         // Same as coverallsapp/github-action
         // @link https://github.com/coverallsapp/github-action/blob/5984097c6e76d873ef1d8e8e1836b0914d307c3c/src/run.ts#L47
         $this->env['CI_JOB_ID'] = $this->env['GITHUB_RUN_ID'];
+        $this->env['CI_BUILD_NUMBER'] = $this->env['GITHUB_RUN_NUMBER'];
```

Comment:
pls check the example number. looking at the doc, it will be just `1` or `2`. we need sth uniq (no other build nor repo repo reporting same number), maybe `GITHUB_ACTION` (`The unique identifier (id) of the action.`)?
maybe just dump the `GITHUB_*` envs in 2 actions and let's see what's there